stdlib/format.ml

```diff
@@ -953,6 +967,18 @@ let default_pp_mark_close_tag = function
 let default_pp_print_open_tag = ignore
 let default_pp_print_close_tag = ignore
 
+let utf8_scalar_width s ~pos ~len =
+  let rec width s count current last =
```

Comment:
nitpick: to me `last` suggest that this position is included, while I read the code as excluding this position.